google/datalab/bigquery/_sampling.py

```diff
@@ -111,3 +111,25 @@ def _random_sampling(sql):
         sql = '%s LIMIT %d' % (sql, count)
       return sql
     return _random_sampling
+
+  @staticmethod
+  def _auto(method, fields, count, percent, key_field, ascending):
+    """Construct a Sorting object according to the provided sampling technique, provided all
+    its needed fields are passed as arguments
+
+    Args:
+      method: sampling strategy, one of the method names above in this module
```

Comment:
This isn't quite right. For instance, `limit` doesn't match `default`.

I would just explicitly list the supported methods.